src/main/resources/db.changelog/changes/booking.yaml

```diff
@@ -0,0 +1,66 @@
+databaseChangeLog:
+  - logicalFilePath: no-path
+  - changeSet:
+      id: booking table init
+      author: basiashvilin@gmail.com
+      preConditions:
+        onFail: MARK_RAN
+        not:
+          tableExists:
+            schemaName:
+            tableName: booking
+      changes:
+        - createTable:
+            tableName: booking
+            columns:
+              - column:
+                  name: bo_id
+                  type: bigint
+                  autoIncrement: true
+                  remarks: 'Booking primary key'
+                  constraints:
+                    primaryKey: true
+                    primaryKeyName: spot_id_pk
+              - column:
+                  name: em_id
+                  type: bigint
+                  remarks: 'Employee foreign key'
```

Comment:
Please add fk to employee

Reply:
i just checked and employee table has been merged to master. foreign key constraint is added.